File: task_for_alexey.java
import java.io.*;
import java.util.*;
class CC
{
    private static long calcAlexTask(long a, long b)
    {
        return(a * b) / gcd(a, b);
    }
 
 
    private static long gcd(long a,long b)
    {
        if(b==0)
        {
            return a;
        }
        else
            return gcd(b,a % b);
    }
   public static void main(String[] args)throws Exception
   {
      Scanner sc=new Scanner(System.in);
      int T;
      T=sc.nextInt();
      while(T-->0)
      {
         int N,i,j,flag=0,count;
         N=sc.nextInt();
         int arr[]=new int[N];
         for(i=0;i<N;i++)
         {
            arr[i]=sc.nextInt();
         }
         
         long sum=Long.MAX_VALUE;
         for(i=0;i<N;i++)
         {
            for(j=i+1;j<N;j++)
            {
                sum=Math.min(sum,calcAlexTask((long)arr[i],(long)arr[j]));
            }
         }
         System.out.println(sum);
      }
   }
}

File: task_for_alexey.cpp
#include<iostream>
using namespace std;
int main()
{
    int T;
    cin>>T;
    while(T--)
    {
        int N,i,j,temp=0,sum=0,sum1=0;
        cin>>N;
        int arr[N];
        for(i=0;i<N;i++)
        {
            cin>>arr[i];
        }
        for(i=0;i<N;i++)
        {
            for(j=i;j<N;j++)
            {
                if(arr[i]>arr[j])
                {
                    temp=arr[i];
                    arr[i]=arr[j];
                    arr[j]=temp;
                }
            }
        }
        sum=0;
        while(true)
        {
            if(sum<sum1)
            {
                sum+=arr[0];
            }
            else
            {
                sum1+=arr[1];
            }
            if(sum==sum1)
            {
                break;
            }
        }
        cout<<sum<<endl;;
    }
}
